extensions/wikia/WAMPage/templates/WAMPageController_index.php

```diff
@@ -1,6 +1,6 @@
 <div class="wam-header wam-hero">
 	<div class="wam-top-gainers-cards">
-		<h2 class="wam-top-gainers-header"><?= wfMessage('wampage-top-gainers-header')->escaped(); ?><? if ( $isSingleVertical ) : ?>: <span class="vertical-<?= Sanitizer::encodeAttribute( $verticalsShorts[ $selectedVerticalId ] ) ?>"><?= $filterVerticals[ $selectedVerticalId ] ?></span><? endif; ?></h2>
+		<h2 class="wam-top-gainers-header"><?= wfMessage('wampage-top-gainers-header')->escaped(); ?><? if ( $isSingleVertical ) : ?>: <span class="vertical-<?= Sanitizer::encodeAttribute( $verticalsShorts[ $selectedVerticalId ] ) ?>"><?= wfMessage( "wam-{$verticalsShorts[$selectedVerticalId]}" )->escaped() ?></span><? endif; ?></h2>
```

Comment:
Please use full name of messages here to allow grepping in the future. In general avoid combining messages dynamically, but if it reasonably improves code quality please name all combinations in comment.
https://www.mediawiki.org/wiki/Localisation#Using_messages